Recrutify/Recrutify.Services/Events/StatusChangeEventProcessor.cs

```diff
@@ -26,7 +26,7 @@ public void Subscribe()
 
         public async Task UpdateCandidatesStatusesAsync(UpdateStatusEventArgs e)
         {
-            var candidates = await _candidateService.GetCandidatesByIdsAsync(e.CandidatesIds);
+            var candidates = await _candidateService.GetCandidatesDTOByIdsAsync(e.CandidatesIds);
```

Comment:
just GetCandidatesByIdsAsync will be enougth

Reply:
+